engine/physics/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_library(Physics)
+module_default_init(Physics)
+
+target_link_libraries(Physics
+    PUBLIC Core
+    PUBLIC Application
+    PUBLIC OldEngine
```

Comment:
Do you really need access to OldEngine? Which dependencies do you have on OldEngine?

Reply:
No, it was a mistake. I updated it now